### ISSUE TITLE
DIRT discrepancies

### ISSUE THREAD

**Issue by tjmahr** [LearningToTalk/L2TDatabase#42]
This issue supercedes the previous discrepancy thread #37, now that there is a single report detailing the discrepancies between the ParticipantInfo and DIRT spreadsheets for all three timepoints and five different cross-sectional stduies.

Links to updated results of this check will be posted as comments below.

The most recent results are always available at [inst/audit/check_integrity.md](https://github.com/LearningToTalk/L2TDatabase/blob/master/inst/audit/check_integrity.md).

We'll keep running this report and fixing errors until everything is consistent. Then we'll need to fix the scores in the database that were imported but later discovered to be incorrect.

**Comment by tjmahr** [comment]
Updated again. [inst/audit/check_integrity.md](https://github.com/LearningToTalk/L2TDatabase/blob/master/inst/audit/check_integrity.md)

**Comment by tjmahr** [comment]
I made a big push to check more columns and fix the remaining discrepancies between the participant info spreadsheets and the data re-entry spreadsheets. We're [all green now. :white_check_mark:](https://github.com/LearningToTalk/L2TDatabase/blob/b13f39cd47689225f7e2394cd3bb0dc70a1f1155/inst/audit/check_integrity.md)

I added dates to the participant info spreadsheet for the BRIEF, CTOPPBlending, CTOPPMemory, CTOPPElision, GFTA, and VerbalFluency, so these are now checked for all studies (except for the Dialect studies). 

The only columns in the DIRT spreadsheets not being checked against a column in a participant info spreadsheet are:

- FruitStroop_Date
- MBCDI_Date
- CDI_Extension_Date
- KBIT_Date

I can tackle the KBIT and CDI dates when I import those scores to the database.
